lib/Logger.js

```diff
@@ -77,13 +86,15 @@ class Logger {
         const rLog = new RequestLogger(Config.logger,
                                        Config.level, Config.dump, Config.end,
                                        unserializeUids(serializedUids));
-        rLog.addDefaultFields({ name: this.name });
+        rLog.setParent(this);
         return rLog;
     }
 
     _doLog(levelName, msg, data) {
         const sLogger = Config.logger;
-        const finalData = { name: this.name, time: Date.now() };
+        const finalData = Object.assign({},
```

Comment:
Should we compute the finalData after checking the shouldLog avoiding lot of useless Object.assign ?
btw: why not using ```objectCopy``` from ```utils```?